staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -309,6 +309,12 @@ if ($tempFile -ne "") {
   $paths += $tempFile
 }
 
+#Collect Nvidia log files
+$tempFile=(CollectLogsFromDirectory -path "C:\AzureData\NvidiaInstallLog" -targetFileName "NvidiaInstallLog-$($timeStamp).zip")
```

Comment:
Are those gpu log files not used when running this script? Compress-Archive will fail if the log files are being written.